dev/release/post-04-website.sh

```diff
@@ -29,6 +29,12 @@ if [ "$#" -ne 2 ]; then
   exit 1
 fi
 
+# Verify archery is available and stop if not
+if archery --help > /dev/null 2>&1; then
```

Comment:
Sorry, I missed `!`...

```suggestion
if ! archery --help > /dev/null 2>&1; then
```